projects/core/src/configurator/commons/connectors/configurator-commons.connector.ts

```diff
@@ -50,6 +51,18 @@ export class ConfiguratorCommonsConnector {
     return this.adapter.addToCart(parameters);
   }
 
+  readConfigurationForCartEntry(
+    parameters: Configurator.ReadFromCartEntryParameters
+  ): Observable<Configurator.Configuration> {
+    return this.adapter
+      .readConfigurationForCartEntry(parameters)
+      .pipe(
+        tap(configuration =>
+          this.configUtilsService.setOwnerKey(configuration.owner)
```

Comment:
Couldn't we give the owner via the Parameters, instead of just the owner key?

Reply:
Yes, maybe better. Then I also don't need to create the owner in the OCC layer